<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentGateWay</class>
 <widget class="QDialog" name="PaymentGateWay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>167</y>
     <width>92</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background-color: #4CAF50; /* Green */
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  cursor: pointer;
border-radius:12px;</string>
   </property>
   <property name="text">
    <string>پرداخت</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>412</y>
     <width>99</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background-color: red; /* Green */
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  cursor: pointer;
border-radius:12px;</string>
   </property>
   <property name="text">
    <string>بازگشت</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>331</y>
     <width>63</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>CVV2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>253</y>
     <width>134</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>شماره کارت</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>292</y>
     <width>53</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>تاریخ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>407</y>
     <width>149</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>مبلغ پرداختی</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>370</y>
     <width>79</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15.5pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>رمز دوم</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>571</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_2"/>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>توسط افراد بعدی پر میشود</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
